scripts/initdb.sql

```diff
@@ -1,4 +1,4 @@
 
-create database facilEspanolDb;
-create user facilEspanolUser with encrypted password 'facilEspanolPass';
-grant all privileges on database facilEspanolDb to facilEspanolUser;
+create database "facilEspanolDb";
+create user "facilEspanolUser" with encrypted password 'facilEspanolPass';
+grant all privileges on database "facilEspanolDb" to "facilEspanolUser";
```

Comment:
not sure if all privilages are needed

Reply:
Hey @rungokarol , can I ask you to provide the privileges you've granted on your local set up then?

Reply:
facilEspanolDb=> \du
                                                      List of roles
       Role name     |                         Attributes                                                         | Member of 
------------------------+---------------------------------------------------------------------------------+-----------
 facilEspanolUser |                                                                                                  | {}
 postgres              | Superuser, Create role, Create DB, Replication, Bypass RLS | {}